doc/howto.rst

```diff
@@ -72,21 +72,25 @@ monitoring data via Linux `perf <https://perf.wiki.kernel.org/index.php/Main_Pag
 kernel subsystem. There are few levels of statistics data being managed by access
 rights. For this demo purposes we collect i915 global metrics, i.e. those
 which describe the whole GPU rather than particular running process. Such
-metrics require CAP_SYS_ADMIN or ``/proc/sys/kernel/perf_event_paranoid<=0``to be
-collected. So, run container as follows to allow GPU metrics collection::
+metrics require CAP_PERFMON or ``/proc/sys/kernel/perf_event_paranoid<=0`` to
+be collected. With older kernels or container runtime versions that lack
```

Comment:
I feel we need to look and describe in more details what "container runtime versions that lack/has the CAP_PERFMON" actually mean. Looking here: https://github.com/docker/docs/pull/15050 it seems that the change to support CAP_PERFMON was added and then reverted due to some incompatibilities. With a brief glance I am really unsure which versions support that and which do not. As such, till we won't have this clarification I think we better leave only CAP_SYS_ADMIN described.

Reply:
We've been using it more than a year with containerd + runc versions available in Ubuntu 20.04 (updates), and it's been working fine.

I read through the docker release notes and googled, but did not find anything definite on it either, and I remember that debugging unsupported caps issues e.g. in Kubernetes environment is a bit of a horror though, so it's OK to keep SYS_ADMIN more prominent.

I think CAP_PERMON should still be mentioned due to security impact (people making services having CAP_SYS_ADMIN would make then nice targets).  I'll try to come up with something tomorrow.